src/main/java/Deadlines.java

```diff
@@ -0,0 +1,13 @@
+public class Deadlines extends Task {
+    private static String by;
+
+    public Deadlines(String taskName, String by) {
+        super(taskName);
+        this.by = by;
+    }
+
+    public String toString() {
+        char done = this.getIsDone() ? 'X' : ' ';
+        return "[D][" + done + "] " + this.getTaskName() + " (by: " + by + " )";
```

Comment:
Maybe this part can be made neater. e.g having "[D]" and "[X]" as 2 different String variables instead of a weird string like this "[D]["